lib/stat_tracker.rb

```diff
@@ -39,12 +39,20 @@ def percentage_home_wins
   
   def percentage_visitor_wins
     visitor_wins = @game_teams.count do |game|
-      game.result == "WIN" && game.hoa == "visitor"
+      game.result == "WIN" && game.hoa == "away"
     end
 
     (visitor_wins / self.total_games).round(2)*100
   end
 
+  def percentage_ties
+    ties = @game_teams.count do |game|
+      game.result == "TIE" && game.hoa == "away"
```

Comment:
Nice catch, since @game_teams represent all games played by both teams if we only check for game.result == "TIE" then our integer output will be twice that of what we want. So, we can limit the input in our method or divide the resultant output by 2.

Reply:
Sounds good! We can refactor that tomorrow.